Cargo.toml

```diff
@@ -6,7 +6,8 @@ members = [
     "operators",
     "k8s/plugin",
     "k8s/proxy",
-    "k8s/supportability"
+    "k8s/supportability",
+    "upgrade-job",
```

Comment:
It's not clear to me what the hierarchy is in this repo, should upgrade-job live in k8s?